
# Velocity:
# In Ackermann mode, this is the velocity of the rover center in m/s
# In Rotation mode, this is the rotational velocity of the rover body in rad/s
float32 velocity
# Signed value, positive on the left
# elevation_rad = arctan(r), where r is the distance to the instantaneous
# center of rotation
float32 elevation_rad

